src/test/unit_tests/jest_tests/test_nc_nsfs_bucket_cli.test.js

```diff
@@ -161,6 +165,44 @@ describe('manage nsfs cli bucket flow', () => {
             path_exists = await is_path_exists(DEFAULT_FS_CONFIG, bucket_temp_dir_path);
             expect(path_exists).toBe(false);
         });
+
+        it('cli delete bucket not fail with force flag, when bucket path is not empty', async () => {
+            let action = ACTIONS.ADD;
+            const bucket_options = { name: "bucket4", owner: bucket_defaults.owner, path: bucket_defaults.path };
+            // write the json_file_options
+            const path_to_option_json_file_name = await create_json_bucket_options(path_to_json_bucket_options_dir, bucket_options);
+            const command_flags = {config_root, from_file: path_to_option_json_file_name};
+            // create the bucket and check the details
+            await exec_manage_cli(TYPES.BUCKET, action, command_flags);
+            const bucket = await read_config_file(config_root, CONFIG_SUBDIRS.BUCKETS, "bucket4");
+            assert_bucket(bucket, bucket_options);
+            //here a dummy file is creating in bucket storage location(bucket_defaults.path), 
+            //using existing config file creation method(create_json_bucket_options) and bucket options code for this
+            await create_json_bucket_options(bucket_defaults.path, bucket_options);
```

Comment:
I think it is confusing to use the bucket option as a dummy file.
Create a separate function for this purpose so it will be clear.

Reply:
used exising code for creating a dummy file, dont wanted add code that do same thing again, I will try to update function name to more generic name

Reply:
I don't see why you need the function for a "dummy" file you can just use ` await fs.promises.writeFile`.